server_src/runners/state.js

```diff
@@ -1655,6 +1651,11 @@ class Runner {
         ""
       );
     }
+    if (!farmPrinters[i].webSocketURL) {
+      farmPrinters[i].webSocketURL = convertHttpUrlToWebsocket(
+        farmPrinters[i].printerURL
+      );
+    }
```

Comment:
Just make sure the webSocketURL key/value actually exists for when people upgrade to this version.